src/main/java/seedu/address/model/person/CurrentPatient.java

```diff
@@ -40,17 +40,17 @@ public String addNoteContent(String content) {
     /**
      * Add referral letter content to the ServedPatient.
      */
-    public String addReferralContent(String content) {
+    public String addReferralContent(String notes, String todayDate, String doctorName) {
```

Comment:
This method signature should be in Referral class instead of ServedPatient class